src/services/copyConfig/policies/policyOptions.js

```diff
@@ -31,158 +31,188 @@ class PolicyOptions extends Options {
-    return this.removeInfo(PolicyOptions.#webSdk, info)
+        tooltip: 'POLICIES_FEDERATION',
+      })
+    }
+    if (response.passwordComplexity) {
+      this.options.branches.push({
+        id: PolicyOptions.#passwordComplexity,
+        name: PolicyOptions.#passwordComplexity,
+        propertyName: PolicyOptions.#passwordComplexity,
+        value: true,
+
+        tooltip: 'POLICIES_PASSWORD_COMPLEXITY',
+      })
+    }
+    if (response.gigyaPlugins) {
+      this.options.branches.push({
+        id: 'gigyaPlugins',
+        name: PolicyOptions.#webSdk,
+        propertyName: 'gigyaPlugins',
+        value: true,
+        tooltip: 'POLICIES_WEBSDK',
+      })
+    }
+    if (response.passwordReset) {
+      this.options.branches.push({
+        id: `${PolicyOptions.#idPrefix}${PolicyOptions.#passwordReset}`,
+        name: PolicyOptions.#passwordReset,
+        propertyName: PolicyOptions.#passwordReset,
+        value: true,
+        tooltip: 'POLICIES_PASSWORD_RESET',
+      })
+    }
+    if (response.profilePhoto) {
+      this.options.branches.push({
+        id: 'profilePhoto',
+        name: PolicyOptions.#profilePhoto,
+        propertyName: 'profilePhoto',
+        value: true,
+        tooltip: 'POLICIES_DEFAULT_PROFILE_PHOTO_DIMENSIONS',
+      })
+    }
+    if (response.registration) {
+      this.options.branches.push({
+        id: PolicyOptions.#registration,
+        name: PolicyOptions.#registration,
+        propertyName: PolicyOptions.#registration,
+        value: true,
+        tooltip: 'POLICIES_REGISTRATION',
+      })
+    }
+    if (response.security) {
+      this.options.branches.push({
+        id: PolicyOptions.#security,
+        name: PolicyOptions.#security,
+        propertyName: PolicyOptions.#security,
+        value: true,
+        tooltip: 'POLICIES_SECURITY',
+      })
+    }
+    if (response.twoFactorAuth) {
+      this.options.branches.push({
+        id: `${PolicyOptions.#idPrefix}TwoFactorAuth`,
+        name: PolicyOptions.#twoFactorAuth,
+        propertyName: `twoFactorAuth`,
+        value: true,
+        tooltip: 'POLICIES_TWO_FACTOR_AUTHENTICATION_PROVIDERS',
+      })
+    }
+    if (response.doubleOptIn) {
+      const id = `${PolicyOptions.#idPrefix}${PolicyOptions.#doubleOptIn}`
+      const doiOption = {
+        id: `${id}`,
+        name: PolicyOptions.#doubleOptIn,
+        propertyName: PolicyOptions.#doubleOptIn,
+        value: true,
+        tooltip: 'POLICIES_DOUBLE_OPT_IN',
+      }
+      doiOption.branches = []
+      if (response.doubleOptIn.nextURL) {
+        doiOption.branches.push({
+          id: `${id}-nextUrl-Link`,
+          name: 'Include Customize Redirection URL',
+          link: `${PolicyOptions.#doubleOptIn}.nextURL`,
+          value: true,
+        })
+      }
+      if (response.doubleOptIn.nextExpiredURL) {
+        doiOption.branches.push({
+          id: `${id}-nextExpiredUrl-Link`,
+          name: 'Include Customize Expired Redirection URL',
```

Comment:
Can we improve this in a way that it doesn't require to have this text directly here? Maybe storing it in "PolicyOptions."?

Reply:
The text was added to the translations file. We are now using translation on this node.
The same was added on email options.